sos/plugins/openstack_tripleo.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2019 Red Hat, Inc., Cedric Jeanneret <cjeanner@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+
+
+class OpenStackTripleO(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+    """Log installation informations from OpenStack Installer
+    """
+
+    plugin_name = 'openstack_tripleo'
+
+    def setup(self):
+        # Notes: recursion is max 2 for container-puppet and tripleo-config
+        # Those directories are present on all OpenStack nodes and
+        # may contain sensitive data (password).
+        self.add_copy_spec([
+                '/var/log/paunch.log',
+                '/var/lib/container-puppet/',
+                '/var/lib/tripleo-config/'
+                ])
```

Comment:
Stylistic nit - sos style for this is to indent one level in, and have the closing brackets on the same indention as the name of the call, E.G.:

```python
self.add_copy_spec([
    '/var/log/paunch.log',
    '/var/lib/container-puppet/'
])
```

Reply:
np - will update, thanks!